### ISSUE TITLE
typeit-react installation error on 2.0.7

### ISSUE THREAD

**Issue by bonesoul** [alexmacarthur/typeit#266]
when i try to upgrade typeit-react 2.0.7 getting this error;

```
npm WARN cleanup Failed to remove some directories [
npm WARN cleanup   [
npm WARN cleanup     'D:\\Coding\\vizesizgezi.com\\frontend\\node_modules\\esbuild',
npm WARN cleanup     [Error: EBUSY: resource busy or locked, rmdir 'D:\Coding\vizesizgezi.com\frontend\node_modules\esbuild'] {
npm WARN cleanup       errno: -4082,
npm WARN cleanup       code: 'EBUSY',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'D:\\Coding\\vizesizgezi.com\\frontend\\node_modules\\esbuild'
npm WARN cleanup     }
npm WARN cleanup   ]
npm WARN cleanup ]
npm ERR! code 1
npm ERR! path D:\Coding\vizesizgezi.com\frontend\node_modules\typeit
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c ./scripts/notice
npm ERR! '.' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
```

2.0.4 seems fine without error, versions including 2.0.5, 2.06 and 2.0.7 throws this error. any ideas?

**Comment by ThykeAdams** [comment]
Hasving the exact same issue here

**Comment by bonesoul** [comment]
also similarly getting

```
npm ERR! code 127
npm ERR! path /src/node_modules/typeit
npm ERR! command failed
npm ERR! command sh -c ./scripts/notice
npm ERR! sh: ./scripts/notice: not found
```

**Comment by alexmacarthur** [comment]
Thanks for reporting this, @bonesoul + @ThykeAdams! It's fixed in v2.0.8:

https://github.com/alexmacarthur/typeit/releases/tag/typeit-react%402.0.8
